data/grc/Arnott_1996_Athenaeus/stemma.gv

```diff
@@ -0,0 +1,23 @@
+# Athenaeus: Arnott 1996
+# Epitome derived from A, contamination from a sibling of A
+
+digraph {
+
+    omega -> A;
+    omega -> sibling_of_A;
+    A -> epitome;
+    sibling_of_A -> epitome [style="dashed"]; # hypothetical ms. derived from the archetype, by definition
+    epitome -> gamma;
+    gamma -> delta;
+    delta -> C;
+    delta -> E;
+    gamma -> Eustathius_ms;
+
+    omega [color="grey", label="ω"]; # Arnott names the archetype α
```

Comment:
Name change ?

Reply:
I first named the archetype `alpha`, but: I thought the interest of having several stemmata for the author was to facilitate comparison, a goal better served by harmonising the name across the stemmata; I saw somewhere you insisted on "l'esprit, plutôt que la lettre".

Do you think this project is primarily about reproducing published stemmata?

Reply:
Ok, but are there both the same, or is there the idea than alpha is the archetype, while omega is the actual original (from which the second hypothetical model of Epitome might or might not be directly derived). Does it make sense ?

Reply:
Yes, indeed, spirit over letter ! In the end, the idea is providing material for any kind of analyses on stemmata, either meta (history of philology) or (for my interest at least) focused on textual history and transmission dynamics !

Reply:
I changed my mind. As you'll see in the updated PR, I went for a solution closer to the source, as the editor makes no explicit statement about the relationship to the archetype. Both *l'esprit* (as far as I can tell from the source text) and *la lettre* (of the slightly unconventional source stemma).

In any case, mining these data to compare stemmata would require a way to select the archetypes, etc.